bindata/v3.11.0/kube-apiserver/recovery-config.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kubecontrolplane.config.openshift.io/v1
+kind: KubeAPIServerConfig
+apiServerArguments:
+  storage-backend:
+  - etcd3
+  storage-media-type:
+  - application/vnd.kubernetes.protobuf
+servingInfo:
+  bindAddress: 0.0.0.0:7443
```

Comment:
no, must be specifically localhost. 127.0.0.1

Reply:
it is secured with certs auth already, but sure